Interface/EmotionalCities.Pluma/EmotionalCities.Pluma.csproj

```diff
@@ -18,13 +18,13 @@
     <PackageOutputPath>..\bin\$(Configuration)</PackageOutputPath>
     <PackageTags>Bonsai Rx Emotional Cities Harp Pluma</PackageTags>
     <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
-    <VersionPrefix>0.1.0</VersionPrefix>
+    <VersionPrefix>0.1.1</VersionPrefix>
     <VersionSuffix></VersionSuffix>
-    <LangVersion>9.0</LangVersion>
```

Comment:
For consistency with the other interface libraries, I would keep this line.

Reply:
5. don't get it if has some to do with Adafruit interface

Reply:
Fix in e6df51338b2275df7aa3e13b3b8a28dbfe264844